src/containers/Register/AddProduct/index.js

```diff
@@ -175,16 +177,11 @@ const AddProduct = ({
           <div className={styles.inputWithButton}>
             <Select
               value={type}
-              style={{ width: "100%" }}
+              style={{ width: '100%' }}
               name="type"
-              onChange={value => handleOnChange({ target: { value, name: 'type' } })}
+              onChange={(value) => handleOnChange({ target: { value, name: 'type' } })}
             >
-              {typesList && typesList.map(item => (
-                <Option value={item.type}>
-                  {item.type}
-                </Option>
-              ))
-              }
+              {typesList && typesList.map((item) => <Option value={item.type}>{item.type}</Option>)}
```

Comment:
Precisamos adicionar a key do react no elemento
```suggestion
              {typesList && typesList.map((item) => <Option key={item.id} value={item.type}>{item.type}</Option>)}
```